src/main/java/bankservice/demo/repository/UserRepository.java

```diff
@@ -0,0 +1,15 @@
+package bankservice.demo.repository;
+
+import bankservice.demo.model.User;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    @Query("select u from users u inner join fetch u.roles where u.id = :id")
+    Optional<User> findByIdFetchRoles(@Param(value = "id") Long id);
+
+    @Query("select u from users u inner join fetch u.roles where u.phoneNumber = :number")
+    Optional<User> findUserByPhoneNumber(@Param(value = "number") String number);
```

Comment:
```suggestion
    Optional<User> findById(Long id);

    @Query("select u from users u inner join fetch u.roles where u.phoneNumber = :number")
    Optional<User> findByPhoneNumber(String number);
```

we can simplify this a bit